src/components/settings/EditOrganizationInformationsDialog.tsx

```diff
@@ -0,0 +1,323 @@
+      }
+    },
+  })
+
+  const getBase64 = (file: Blob) => {
+    var reader = new FileReader()
+
+    if (file.size > FILE_MAX_SIZE) {
+      setLogoUploadError(true)
+      return
+    }
+
+    reader.readAsDataURL(file)
+    reader.onload = function () {
+      setLogo(reader?.result?.toString())
+    }
+    reader.onerror = function (error) {
+      // eslint-disable-next-line no-console
+      console.error('Error: ', error)
+    }
+  }
+
+  useEffect(() => {
+    mounted.current = true
+
+    return () => {
+      mounted.current = false
+    }
+  }, [])
+
+  return (
+    <Dialog
+      ref={ref}
+      title={translate('text_62728ff857d47b013204c76c')}
+      description={translate('text_62728ff857d47b013204c77a')}
+      actions={({ closeDialog }) => (
+        <>
+          <Button
+            variant="quaternary"
+            onClick={() => {
+              closeDialog()
+              if (mounted.current) {
+                formikProps.resetForm()
+              }
+            }}
+          >
+            {translate('text_62728ff857d47b013204c7e4')}
+          </Button>
+          <Button
+            variant="primary"
+            disabled={!formikProps.isValid}
+            onClick={async () => await formikProps.handleSubmit()}
+          >
+            {translate('text_62ab2d0396dd6b0361614db2')}
+          </Button>
+        </>
+      )}
+    >
+      <Content>
+        <FormSection>
+          <AvatarContainer>
+            {logo || organization?.logoUrl ? (
+              <Avatar size="large" variant="connector">
+                <img
+                  src={(logo || organization?.logoUrl) as string}
+                  alt={`${organization?.name}'s logo`}
+                />
+              </Avatar>
+            ) : (
+              <Avatar
+                size="large"
+                variant="company"
+                identifier={organization?.name || ''}
+                initials={(organization?.name || '')
+                  .split(' ')
+                  .reduce((acc, n) => (acc = acc + n[0]), '')}
+              />
+            )}
+            <AvatarUploadWrapper>
+              <Button variant="secondary" onClick={() => hiddenFileInputRef?.current?.click()}>
+                {translate('text_62ab2d0396dd6b0361614d18')}
+              </Button>
+              <Typography variant="caption" color={logoUploadError ? 'red' : ''}>
+                {translate('text_62ab2d0396dd6b0361614d20')}
+              </Typography>
+            </AvatarUploadWrapper>
+            <input
+              type="file"
+              accept="image/png, image/jpeg"
+              ref={hiddenFileInputRef}
+              onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+                setLogoUploadError(false)
+                const file = event?.target?.files?.[0]
+
+                if (file) {
+                  getBase64(file)
+                }
+              }}
+              style={{ display: 'none' }}
```

Comment:
We avoid inline style, all style should be defined in a stlyed-component def

Reply:
Removed